drivers/usb/device/usb_dc_stm32.c

```diff
@@ -75,7 +75,7 @@ static const struct soc_gpio_pinctrl usb_pinctrl[] =
  * WARNING: Don't mix USB defined in STM32Cube HAL and CONFIG_USB from Zephyr
  * Kconfig system.
  */
-#ifdef USB
+#if defined(USB) || defined(USB_DRD_FS)
```

Comment:
Could you please update the comment above, explaining what USB_DRD_FS represents?

Reply:
"USB dual role device full speed" IIRC, that's defined by ST here:
https://github.com/zephyrproject-rtos/hal_stm32/blob/main/stm32cube/stm32g0xx/soc/stm32g0b0xx.h#L701

Reply:
yes but update the comment above since it only mention USB and USB_OTG_FS types.

Reply:
Got it